docs/manual/web_framework_examples.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE HTML>
```

Comment:
This file seems entirely new although not related to the changes made here, any idea why it's getting generated like this?

Reply:
I think this is because @isabelatkinson added the info to the manual in #683 but we did not actually regenerate the manual files at that time?